tests/kernel/qa_test_klp.pm

```diff
@@ -30,9 +30,11 @@ sub run {
     my ($test_type) = $git_repo =~ /qa_test_(\w+).git/;
 
     # Set and check patch variables
-    my $incident_id = get_var('INCIDENT_ID');
-    my $patch       = get_var('INCIDENT_PATCH');
-    check_patch_variables($patch, $incident_id) if (is_sle && !get_var('BETA'));
+    unless (get_var('KOTD_REPO') || !(is_sle && !get_var('BETA'))) {
+        my $incident_id = get_var('INCIDENT_ID');
```

Comment:
The condition is IMHO too complicated. Can't you at least use `if`? Or `!(is_sle && !get_var('BETA'))` could be just `(!is_sle || get_var('BETA'))`.

Reply:
i I thinking about this  we need  `not (..&& not ..) ` , `(not .. && ..)` has different meaning

Reply:
I was just concern about number of negations: 3 (`unless` and 2x `!`). I guess instead of `not (..&& not ..)` we can have `(.. || ||)`.
So how about `if ((is_sle && !get_var('BETA')) || !get_var('KOTD_REPO'))` ?

i.e. just adding `!get_var('KOTD_REPO')` to the original condition looks much readable to me, but I'm not a perl developer :). Consider approved anyway :)

Reply:
And I'd put a comment there, that `(is_sle && !get_var('BETA'))` means QAM SLES releases (i.e. no future SLES or any openSUSE).

Reply:
More and more thinking about removing this part of code.... Oiginal purpose vas different than only check existing patch. And test is started only after install_ltp with all needed things set and check .